kubernetes/gatekeeper.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Not for this PR necessarily, but there's currently quite the variety of shebangs in `kubernetes/`:

```==> kubernetes/curriculum.sh <==
#!/bin/sh

==> kubernetes/gatekeeper.sh <==
#!/bin/bash

==> kubernetes/launch-common.sh <==
#!/bin/bash -e

==> kubernetes/launch-job.sh <==
#!/bin/bash -eu

==> kubernetes/launch-match.sh <==
#!/bin/bash -e

==> kubernetes/log-git-commit.sh <==
#!/bin/bash -e

==> kubernetes/match.sh <==
#!/bin/bash

==> kubernetes/shuffle-and-export.sh <==
#!/bin/bash -e

==> kubernetes/train.sh <==
#!/bin/bash -e

==> kubernetes/victimplay-predictor.sh <==
#!/bin/sh

==> kubernetes/victimplay.sh <==
#!/bin/sh
```

I think we should probably standardize on bash vs sh? And I lean towards using `-e` where possible.